engines/ultima/configure.engine

```diff
@@ -0,0 +1,6 @@
+# This file is included from the main "configure" script
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
+add_engine ultima "Ultima games" no "ultima1 ultima6 ultima8"
+add_engine ultima1 "Ultima I" no "" "" "highres 16bit"
```

Comment:
> Also, while we have you here ;) Dreammaster needed any disassemblies which you probably still have, so he could complete the support for other sub-engines.

I hope you don't take it that I'm stepping on your toes, so to speak. If you intend to keep working on the games, you'd remain the primary developer. And goodness knows I have plenty to get on with for the other games of the series. But it would be a good idea, if you're willing, to make a copy of your disassemblies available to us, Dominus, and any others, just for archival purposes if nothing else.

Reply:
> quick question. What is the reason for having individual games as the targets? I mean, usually we do it only when there are some difference in hardware requirements like resolution.

Now that I think of it, that's actually a good question. My initial though was that the sub-engines are more or less full blown engines in their own right, just consolidated together in one overall engine, with some shared code for event handling, saves, screen, etc. This is moreso than engines like TsAGE where although the different games have their own hardcoded logic, they share more of the engine in common. But in retrospect, you're right.. the sub-engines aren't so big that I can see a particular need or advantage for having them individually disable-able. I'll go ahead and remove them. This will also simplify the module.mk

> Also, is ultima1 really highres and 16bit?

Not in classic mode, but it also has even-more-in-progress code for creating an enhanced mode that will use the Ultima 6 tileset. Though if there won't be individual sub-engines listed anymore, it's a moot point.

Reply:
@sev- Yes I've got some IDA dissasseblies for U6 and MD. Where should I put them?

Reply:
Please e-mail them to me at sev@scummvm.org or upload to thing like Google Drive and share the link